### PR TITLE
[Event Hubs Client] Track Two (Formatting Pass)

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to run an automated formatting pass over each of the three Event Hubs solutions.   No functional or orthogonal changes were intended.

# Last Upstream Rebase

Monday, December 9, 9:33am (EST)

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducerClient/EventHubProducerClientTests.cs [hunk 1]
```diff
@@ -594,14 +598,17 @@ public async Task CloseAsyncSurfacesExceptionsForTransportConsumers()
         {
             var mockTransportProducer = new Mock<TransportProducer>();
             var mockConnection = new MockConnection(() => mockTransportProducer.Object);
-            var mockBatch = new EventDataBatch(new MockTransportBatch(), new SendEventOptions { PartitionId = "1" });;
+            var mockBatch = new EventDataBatch(new MockTransportBatch(), new SendEventOptions { PartitionId = "1" });
+            ;
```
[the remark this answers is not in the document]
Comment (kinelski) [reply sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducerClient/EventHubProducerClientTests.cs]: We should remove the extra `;`, by the way.

**Review by kinelski — APPROVED** [review]
LGTM

**Review by ShivangiReja — APPROVED** [review]

File: sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientLiveTests.cs [hunk 2]
```diff
@@ -934,11 +934,11 @@ await using (var connection = new EventHubConnection(connectionString))
...
-                            // TODO: fix test. OwnerIdentifier is not accessible anymore.
-                            // onInitialize: eventArgs =>
-                            //     ownedPartitionsCount.AddOrUpdate(eventArgs.Context.OwnerIdentifier, 1, (ownerId, value) => value + 1),
-                            // onStop: eventArgs =>
-                            //     ownedPartitionsCount.AddOrUpdate(eventArgs.Context.OwnerIdentifier, 0, (ownerId, value) => value - 1)
+                        // TODO: fix test. OwnerIdentifier is not accessible anymore.
+                        // onInitialize: eventArgs =>
+                        //     ownedPartitionsCount.AddOrUpdate(eventArgs.Context.OwnerIdentifier, 1, (ownerId, value) => value + 1),
+                        // onStop: eventArgs =>
+                        //     ownedPartitionsCount.AddOrUpdate(eventArgs.Context.OwnerIdentifier, 0, (ownerId, value) => value - 1)
```
[the remark this answers is not in the document]
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientLiveTests.cs]: yeah... I agree that's weird.   I restored the spaces for alignment.

File: sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Resources.Designer.cs [hunk 3]
```diff
@@ -8,9 +8,8 @@
 // </auto-generated>
```
[the remark this answers is not in the document]
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Resources.Designer.cs]: I hate it as well, but not really, unless we wanted to write our own tool.   The tool itself plays by its own rules.   We're also doing some hand-editing due to the sharing and rules around embedded resource namespacing which throw things off.

File: sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducerClient/EventHubProducerClientTests.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducerClient/EventHubProducerClientTests.cs]: Yes, it sure did try.    Removed.

**Pull request merged**